app/views/admin/shared/_tooltip_button.html.haml

```diff
@@ -0,0 +1,7 @@
+%div{ "data-controller": "tooltip" }
+  %button{class: button_class, "data-reflex": button_reflex, "data-id": reflex_data_id, "data-tooltip-target": "element" }
+  .tooltip-container
+    .tooltip{"data-tooltip-target": "tooltip"}
+      = sanitize tooltip_text
```

Comment:
Would it not be sanitised automatically?

Reply:
I believe rails escape html automatically but does not sanitize unless you do it manually.